bundles/plugins/org.bonitasoft.studio.businessobject/src-test/java/org/bonitasoft/studio/businessobject/core/expression/QueryExpressionProviderTest.java

```diff
@@ -76,7 +68,8 @@ public void should_buildQueryExpressionModel_return_QueryExpressionModel_from_bo
         assertThat(model.getBusinessObjects()).hasSize(1);
         final BusinessObjectExpressionQuery businessObjectExpressionQuery = model.getBusinessObjects().get(0);
         assertThat(businessObjectExpressionQuery.getQualifiedName()).isEqualTo("org.bonita.Employee");
-        assertThat(businessObjectExpressionQuery.getQueryExpressions()).hasSize(4);
+        assertThat(businessObjectExpressionQuery.getQueryExpressions()).extracting("name").contains("Employee.find", "Employee.findByEmployeeId",
```

Comment:
previously, we checked that they are 4 elements but now we are checking for 3 names. Shoudln't we check for 4 (or 5 if I understood well and a new request has been added) also or put the 4( or5) names and use a containsExactly?